docs/how-to-guides/deploy.md

```diff
@@ -6,22 +6,78 @@ id: deploy
 
 ## Overview
 
-This guide shows you how to deploy any realm or package to the Gno chain. Deployment is be done by utilizing `gnokey`'s `maketx addpkg` API.
+This guide shows you how to deploy any realm or package to the Gno chain. It will
+show you how to:
+- Deploy Gno code in your browser via the Playground, 
+- Deploy Gno code from your local machine using `gnokey`'s `maketx addpkg` API.
+
+
+## Deployment via the Playground
+
+Deployment via the Playground is recommended for smaller realms and packages.
+For larger projects, it is recommended to write, test, and deploy your code from
+a local environment.
+For this, check out [**Deployment from a local environment**](#deployment-from-a-local-environment).
+
+### Prerequisites
+
+- **Internet connection**
+- **A keypair in a Gno.land wallet, such as [Adena](https://adena.app)**
+
+### Using Gno Playground
+
+You can write, test, and deploy packages and realms using Gno Playground.
+To start using the Playground, you can check out XYZ.
```

Comment:
Is the 'XYZ' a placeholder?

Reply:
Yup, removed it. [e53b6ac](https://github.com/gnolang/gno/pull/1766/commits/e53b6acc2b0695196660a9a19994c66edbe43191)